src/PlaneWaveBasis.jl

```diff
@@ -229,10 +232,12 @@ treated explicitly. In this case all guess densities and potential
 functions must agree with the crystal symmetries or the result is
 undefined.
 """
-function PlaneWaveBasis(model::Model, Ecut::Number;
+function PlaneWaveBasis(model::Model,
+                        Ecut::Union{Number, Quantity};
                         kgrid=kgrid_size_from_minimal_spacing(model.lattice, 2π * 0.022),
                         kshift=[iseven(nk) ? 1/2 : 0 for nk in kgrid],
                         use_symmetry=true, kwargs...)
+    kgrid = austrip.(kgrid)
```

Comment:
Hmm. I don't think this makes sense. `kgrid` is in either case just a number. (In fact just a vector of three integers). I suppose you wanted to add units to the kgrid spacing, right? That should go to the `kgrid_size_from_minimal_spacing` function.

Reply:
That means that if you don't use the default argument, you won't be able to pass an explicit value for the kgrid with units.

Reply:
Kgrid is int, there's no units there

Reply:
Yes sorry, that's what I meant. Only the `kgrid_size_from_minimal_spacing` (which takes a kgrid spacing as second argument) needs to get some extension to support units.

Reply:
I still don't understand this, but I'll change it.